.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,34 @@
+name: Python package
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [2.7, 3.5, 3.6, 3.7]
```

Comment:
It will start 4 different VMs. Do we need 4 different versions for linting and test? I would use one py 3.7 for this ci.

@amanbha @yaron2 @brendandburns Do we expect that dapr py-sdk supports python 2.x ?

Reply:
I will change it to 3.7 only.  
I agree @youngbupark , since python 2.x is retiring in 1 month 23 days from now , it will be better to remove 2.x .